### ISSUE TITLE
Megathread of all MycetOS Fenarimol compounds – future, current, finished and halted compounds

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#29]
Following our meeting on 29th April 2020, I will start this issue thread where we can have an up-to-date list of fenarimol compounds, until we can find a better solution.
@bendndi Can you put a comment below this with the most up-to-date list of compound progress? And can you please update this list by replacing the image with a newer version of the image instead of making a new comment (this is to reduce clutter on this thread).
Please limit comments to this thread to update on your progress only. That is:

- Compounds you would like to synthesise, either taken from the unshaded compounds in the current list or ones of your own design (so Ben can add them into the list as light orange)
- Compounds you successfully synthesised (so that Ben can colour them green)
- Compounds you have stopped working on due to difficulties (so Ben can mark those as dark orange)

To reduce clutter, **no suggestions, recommendations, discussions are to be posted here, only synthetic progress**. Please send those to the other threads in this repositories such as Sevenoaks thread, the Open Synthesis Network thread and Hung's PhD thread.

The order of things to appear in this thread is as follows:
- This comment you are reading at the top
- Ben's comment with the most up-to-date list of compounds (to be updated as new materials emerge)
- Everyone's progress below as comments.

@bendndi What is the best way to help you compile this list with minimal effort? Should be give you the structures as an image? Should we give you the names of the compounds, SMILES strings etc? Also state which organisations (unis/schools etc) are working on the compounds? Can you let us know below? I will update that information here.

**Comment by bendndi** [comment]
Up to date Fenarimol list:
![image](https://user-images.githubusercontent.com/36627399/81159060-e182ee80-8f88-11ea-9f1c-1412332417eb.png)

**Comment by bendndi** [comment]
Thanks @fantasy121 

To help me keep this file updated, please use the following guidelines:

**To notify me of componuds you plan to synthesize** please make a post in which you list the compounds as either:
- the Target ID from the image above (if the compound already exists in the list)
or 
- as a SMILES string (if it is a new suggestion not currently in the list above).
- Also please note your location / name / institution 

**If you have completed synthesis on a compound** from the above list:
- please post the Target ID and tag as "successfully synthesized"
- If you have successfully synthesized a componud which is not present in the list, please list as a SMILES string
- If there is an internal / institutional identifier for the compound (e.g. a lab notebook reference number or a company ID) please also include this

**If you have run into issues and believe the compound cannot be readily synthesized** 
- please post the Target ID and tag as "difficulties in synthesis"

**Comment by bendndi** [comment]
To help me get the list above up to date, can everyone please post below the componuds they have successfully synthesized, are currently working on, or would like to add to the target list (ideally as smiles strings but in this first instance a single image of structures is also fine if this is less work on your side)
@fantasy121 @dmitrij176 @mattodd @mattodowd25

**Comment by dmitrij176** [comment]
![Synthesis update May 2020 ](https://user-images.githubusercontent.com/44785501/81188746-650b0280-8fbe-11ea-882f-b3d230a6c6ce.jpeg)

**Comment by bendndi** [comment]
@dmitrij176  Any chance you're able to share this as a SMILES list e.g. an excel table with smiles column, status colum and ID column? It is much easier and quicker to handle SMILES than having to resketch compounds by had each time.
(I can help you get the SMILES strings if you can tell me which drawing package you're using)

**Comment by fantasy121** [comment]
![20200519 11 Compounds  GitHub Update Format](https://user-images.githubusercontent.com/36176821/82277563-865eec00-99cb-11ea-8ef7-3371181391ab.png)

Name: Hung Phat Duong
Institution: University of Sydney
Location: Australia

_Already on OSN list:_
Done: 
HPD035-1P6 — P4_D_MXF004
HPD037-1 — P4_D_MXF005

Scheduled:
HPD042-1 — P4_D_MXF006

Difficult (Stopped)
HPD036-2 — P4_D_MXF009

_Not on OSN list:_

Done:
HPD033-3P3 — BrC(C=C1)=CC=C1C(C2CCCCC2)(O)C3=CN=CC=C3
HPD033-2P4 — OC(CCCC)(C1=CC=C(Br)C=C1)C2=CN=CC=C2

Scheduled:
HPD034-4 — OC(C1CCCC1)(C2=CC=C(Br)C=C2)C3=CN=CC=C3
HPD031-2 — ClC(C=C1)=CC=C1C(C2=CC=CC=C2)(O)C3=CC=CC=C3Br
HPD032-2 — ClC(C=C1)=CC=C1C(C2CCCCC2)(O)C3=CC=CC=C3Br
HPD041-2 — OC(C1=CC=C(Br)C=C1)(C2=CC=C(Cl)C=C2)C3=CC=CC=C3
HPD043-1 — OC(C1=CC=C(Br)C=C1)(C2=CC=C(Cl)C=C2)C3=CN=CC=C3

@bendndi Can you change the "synthesised by" line for me from "HDong" to "HDuong", it's missing a "u". Thanks.

**Comment by wwjvdsande** [comment]
@fantasy121 . We received the compounds for screening. Could you sent us the molecular weights so that we can make the proper dilutions for screening? We could screen them tomorrow if we have this informaiton

**Comment by wwjvdsande** [comment]
@dmitrij176 we are screening your compounds as well. Hopefully we have some update soon on the activity of these compounds. We are now screening a high and low dose to see which are the most potent ones.

**Comment by mattodd** [comment]
Hi @wwjvdsande it's great to hear you're back and in lab. Thanks for the update and looking forward to those new data! Hope all goes well.

**Comment by wwjvdsande** [comment]
Hi @mattodd yes we have started up slowly. Not all systems are running yet, but the in vitro susceptibility testing has started. We are not back full-time yet but are starting up the most essential experiments.

**Comment by MFernflower** [comment]
@mattodd @fantasy121 great news!!!!

Hopefully we can get more fungal potency data and perhaps get the ameobae screen done by fall!

**Comment by bendndi** [comment]
@fantasy121  I'm adding in your compounds from [this post](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/29#issuecomment-630538844) and for HPD033-3P3 the image shows it as a chlorophenyl but the smiles shos it as a bromophenyl - could you please confirm which one it is?

**Comment by bendndi** [comment]
List updated to reflect most recent changes

**Comment by fantasy121** [comment]
> @fantasy121 I'm adding in your compounds from [this post](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/29#issuecomment-630538844) and for HPD033-3P3 the image shows it as a chlorophenyl but the smiles shos it as a bromophenyl - could you please confirm which one it is?

I've corrected my drawings. The SMILES were correct. Also for HPD34-4 as well as theseare related

**Comment by MFernflower** [comment]
@fantasy121 Any lab updates?

**Comment by MFernflower** [comment]
@fantasy121 I see you have been having issues oxidizing a precursor alcohol to it's ketone using manganese - have you tried Hypervalent Iodine oxidants like PIDA or DMP?

**Comment by MFernflower** [comment]
@dmitrij176 I found a possibly useful paper when doing research into perfumery: https://orgmedchemlett.springeropen.com/track/pdf/10.1186/2191-2858-1-12

facile synthesis of oximes using solvent free methods

**Comment by fantasy121** [comment]
> @dmitrij176 I found a possibly useful paper when doing research into perfumery: https://orgmedchemlett.springeropen.com/track/pdf/10.1186/2191-2858-1-12
> 
> facile synthesis of oximes using solvent free methods

Thanks @MFernflower I had DMP as a potential alternative should MnO2 oxidation not working. Benefit of MnO2 is that it is quite cheap, readily available and easy to handle. I actually had it working for me quite well a few years back but am unsure as to why it stops working now. So currently doing some testing to see whether it's the problem with the starting ketones or with the  MnO2 reagent itself. Testing old ketone vs new ketones, and testing my bottle of MnO2 vs borrowed MnO2 from another group. I will also be ordering some new activated MnO2 to see if that solves my problem.

**Comment by MFernflower** [comment]
@fantasy121 it appears the ketone is commercially available under the
unusual moniker of 3-Benzoylpyridine

Might be worth buying to avoid the annoying task of oven drying inorganic
reagents!

**Comment by fantasy121** [comment]
> @fantasy121 it appears the ketone is commercially available under the unusual moniker of 3-Benzoylpyridine Might be worth buying to avoid the annoying task of oven drying inorganic reagents!

Thanks @MFernflower . Great suggestion. I think I initially avoided this option as I expected the conversion to be easy (from personal time experience) but if this one keeps giving me problems definitely I will get it commercially.

**Comment by MFernflower** [comment]
@fantasy121 triphenylmethanol might not be a bad thing to send with the two other compounds that have had the pyridine removed to wendy

**Comment by MFernflower** [comment]
@kym834 I hate to be _that guy_ but would it be possible to post an overview of the fenarimol compounds you will be making?

**Comment by kym834** [comment]
Hi All, From the list above I've made plans to synthesis the following compounds and I would like to put my name down for them
P4_A_009
P4_A_010
P4_B_008
P4_B_011
P4_B_012

Sydney, Australia / Kymberley Scroggie / University of Sydney

**Comment by bendndi** [comment]
Thanks @kym834  - list at top of page updated to reflect this.

**Comment by dmitrij176** [comment]
<img width="329" alt="image" src="https://user-images.githubusercontent.com/44785501/93774783-8856e700-fc19-11ea-99b9-5a8e6e0ac11b.png">

Synthesis update:
The two analogs DM27-1 and DM28-1 have been successfully synthesised. @bendndi could you please update the OSN list.

**Comment by fantasy121** [comment]
![20200921ProposedBrBzRxn Products](https://user-images.githubusercontent.com/36176821/93834137-1e444d80-fcbe-11ea-8ce4-e740d3a97bba.jpg)
Additional compounds that I'm planning to make.

I will also claim P4_D_MXF007 from the OSN list

Typo correction from previous update.
![20200519 11 Compounds  GitHub Update Format](https://user-images.githubusercontent.com/36176821/93834193-55b2fa00-fcbe-11ea-9d31-010ce523e264.jpg)
HPD033-2P4 should contain an extra Br- as depicted 
Corrected SMILES string: OC(CCCC)(C1=CN=CC=C1)C2=CC=C(Br)C=C2

**Comment by MFernflower** [comment]
@fantasy121 If i'm not mistaken ripping off the pyridine ring will cause the compound to become inactive but hey more data is more data

**Comment by bendndi** [comment]
@fantasy121  @MFernflower  Indeed, pretty much the only thing we can say for certain about the SAR of fenarimols is the losing that 3-pyridiyl will remove CYP51 inhibition and therefore is hgihly likely (read: almost certain) to kill any anti-fungal activity. i'd mkae one example to confirm this, or I'd make the hindered 2-methyl or 6-methyl pyrdine analogue to prove this point, but I wouldn't prioritize a suite of such compounds.

**Comment by MFernflower** [comment]
@bendndi @fantasy121  Indeed - removing the nitrogen prevents the molecule from binding the iron atom in the cyp51 active site

![cyp51_active_site](https://user-images.githubusercontent.com/3164942/93962073-e4a82c80-fd27-11ea-9009-e30689882987.png)

**Comment by mattodd** [comment]
Agreed @bendndi. @fantasy121 I'd make one Ph analog for now and combine the joined-ring analogs with the Pyr.

**Comment by MFernflower** [comment]
@fantasy121 looks good - might I ask if you could try reacting dicyclohexylketone with the pyridine lithium?

**Comment by dmitrij176** [comment]
![Ghana OSN compounds](https://user-images.githubusercontent.com/44785501/96347240-324e5580-1098-11eb-8550-c3d0033f36e5.jpeg)

@wwjvdsande @mattodd @bendndi 10 compounds from the University of Ghana along with my 2 novel analogues DM27-1 and DM28-1 were dispatched to Rotterdam yesterday. @wwjvdsande they are scheduled for delivery on Monday, so in case of any post issues please let me know.

**Comment by dmitrij176** [comment]
@wwjvdsande compound coding error in A06 and A07 has been resolved and labels changed accordingly to match the MW values in the attached scheme.

**Comment by fantasy121** [comment]
> @fantasy121 looks good - might I ask if you could try reacting dicyclohexylketone with the pyridine lithium?

I'll get to it once this batch is done as I need to order it and it might take a bit of time. Reactants for the ones I'm planning to do now are more accessible. Will update when I get to that point.

**Comment by bendndi** [comment]
> ![2020OctNov - GitHub propose pt 2](https://user-images.githubusercontent.com/36176821/96214475-46765200-0fc7-11eb-8e9c-1ab435cca4be.jpg)
> Updated list of compounds I'm planning to make. Removing non-pyridyl compounds apart from a small batch I completed. Looking more into aliphatic ring conversion which reduces LogD values (inspiration from HPD033-3P3).

Hi @fantasy121 , would it be possible for you to send me this list as a table of SMILES strings like below? Makes it so much easier than uploading each structure by hand (30 seconds instead of 30 minutes)

![image](https://user-images.githubusercontent.com/36627399/96558235-aa489580-12bb-11eb-9a2a-f979bdb1cec0.png)

**Comment by bendndi** [comment]
Updated the OSN list to reflect UofGhana compiunds. Will update again with new molecules from @fantasy121 if I can get the smiles (may take longer for me to get around to it if I update by hand)

**Comment by Wilson-Lm** [comment]
Hi @dmitrij176 
We have received your compounds! It arrived yesterday as mentioned. Also noted on the compound coding error for A06 and A07. 

@fantasy121 @dmitrij176 One question which is also wise to have a think before our next meeting, 
Do we want to standardise the names for the compounds that were synthesised for the 2020 fenarimol manuscript? 
We currently have HPD-, DM- and LR-. Would it be easier if they have a similar coding system? What do you all think?

**Comment by bendndi** [comment]
@Wilson-Lm Typically in a manuscript with a large range of compounds such as this one, the `standard` approach is to just number to componuds rather than use trivial names. e.g. 1, 2, 3...

**Comment by bendndi** [comment]
We did discuss earlier about have a standardised MycetOS numbering system. I still believe this is a good idea (and should be a standardised process across the project). There was an issue thread on this at the start of MycetOS but I can no longer see it.

**Comment by mattodd** [comment]
@bendndi @Wilson-Lm YES PLEASE, we need to use the standardised numbering system. Internal codes are fine internally, but not when we start sending compounds around for evaluation and discussing them here together. The numbering method is [here](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention), and needs to be used in conjunction with the [Master List](https://docs.google.com/spreadsheets/d/1YhK-3i2KwuVabo1GbZSgVjAUbavEICCMKi5v-EhNq80/edit#gid=518064767&vpid=B2), so that compounds do not receive duplicate numbers. It looks like we have not numbered any of these compounds properly, is that right @fantasy121 ? In which case let's make a start with your compounds up the top of the sheet.

**Comment by Wilson-Lm** [comment]
_"compound coding error in A06 and A07 has been resolved and labels changed accordingly to match the MW values in the attached  @scheme."_ 

@dmitrij176 , I just want to double confirm the compounds before reconstituting and discarding the original bottle. is the information below correct?
A06 - mass, 68.68mg; MW, 319.13
A07 - mass 38.03; MW, 347.18

**Comment by dmitrij176** [comment]
> _"compound coding error in A06 and A07 has been resolved and labels changed accordingly to match the MW values in the attached @scheme."_
> 
> @dmitrij176 , I just want to double confirm the compounds before reconstituting and discarding the original bottle. is the information below correct?
> A06 - mass, 68.68mg; MW, 319.13
> A07 - mass 38.03; MW, 347.18

Hi Wilson.
Yes, thats right. I have double checked all the samples and can confirm that there is a full match between all data points.

**Comment by Wilson-Lm** [comment]
Thank you! @dmitrij176

**Comment by wwjvdsande** [comment]
> @bendndi @Wilson-Lm YES PLEASE, we need to use the standardised numbering system. Internal codes are fine internally, but not when we start sending compounds around for evaluation and discussing them here together. The numbering method is [here](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention), and needs to be used in conjunction with the [Master List](https://docs.google.com/spreadsheets/d/1YhK-3i2KwuVabo1GbZSgVjAUbavEICCMKi5v-EhNq80/edit#gid=518064767&vpid=B2), so that compounds do not receive duplicate numbers. It looks like we have not numbered any of these compounds properly, is that right @fantasy121 ? In which case let's make a start with your compounds up the top of the sheet.

@mattodd @bendndi @fantasy121 @dmitrij176 @Wilson-Lm is there a list were the "renumbering of the old names" into the official names is deposited? For us non-chemists it is a bit difficult to do the renumbering. We need to add the official names to the figures and tables when drafting the manuscript. We currently still use the numbers on the compounds received.

**Comment by dmitrij176** [comment]
Good day @bendndi and @mattodd. I am writing to inform you that the work on novel Fenarimol coding in the Master List is  complete. All molecules have been assigned with unique codes. I am just reconfirming few entries with @fantasy121. I want to make sure that everything is correct before project members start to use MYOS numbering in research papers, drafts etc.
The only thing which I wasnt sure was the presence of Cetirizine.HCl in the MYOS Compounds List. The mentioned molecule is a marketed antihistamine drug but it looks like it has been tested previously. Has it been resynthesised and then tested? Finally, is it supposed to be in that list if its not novel?
Can someone please clarify.
Thank you in advance

**Comment by fantasy121** [comment]
@dmitrij176 Mat brought to my attention Cetirizine was available for purchase and we agreed it would be interesting to see its activity. This compound was evaluated but was not synthesised by me. In the context of mycetoma, Cetirizine is novel in the sense that it hasn't been tested before, but not that it is a new molecule that hasn't been synthesised before. I think all information is good information and we should have Cetirizine in with the data as well.
